packages/garbo/src/tasks/barfTurn.ts

```diff
@@ -490,6 +490,21 @@ const NonBarfTurnTasks: AlternateTask[] = [
   },
 ];
 
+function canDartsGood(ready: boolean): boolean {
+  const havePerks =
+    get("everfullDartPerks").includes("25% Better bullseye targeting") &&
+    get("everfullDartPerks").includes("25% More Accurate bullseye targeting");
+  const haveItems = havePerks
+    ? have($item`Everfull Dart Holster`)
+    : have($item`Everfull Dart Holster`) && have($item`spring shoes`);
+
+  if (ready) return haveItems;
```

Comment:
This chunk of code is totally unrelated to the rest of the function. Should these actually be one function?

Reply:
Good call. It should really be three consts.